lib/DxilValidation/DxilValidation.cpp

```diff
@@ -2443,7 +2443,15 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
   case DXIL::OpCode::VectorAccumulate:
 
     break;
-
+  case DXIL::OpCode::IsInf:
+  case DXIL::OpCode::IsNaN:
+  case DXIL::OpCode::IsFinite:
+  case DXIL::OpCode::IsNormal: {
+    if (!ValCtx.DxilMod.GetShaderModel()->IsSM69Plus() &&
+        CI->getOperand(1)->getType()->getScalarType()->isHalfTy())
```

Comment:
I'm assuming no but just want to make sure. There error doesn't raise on half-emulated-as-float code right?